onchain/rollups/contracts/library/LibProof.sol

```diff
@@ -0,0 +1,21 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+pragma solidity ^0.8.8;
+
+import {Proof} from "../common/Proof.sol";
```

Comment:
I'd like to decouple those things.

Reply:
Suppose you use `Proof` but not `LibProof`. If they live on the same file, and I make a change in `LibProof`, even if `Proof` stays the same, this might trigger a redeployment for you, right?